modules/custom/az_core/az_enterprise_attributes_import/az_enterprise_attributes_import.install

```diff
@@ -22,3 +22,10 @@ function az_enterprise_attributes_import_install() {
   }
 
 }
+
+/**
+ * Enable migrate_queue_importer module for dependency.
+ */
+function az_enterprise_attributes_import_update_1020801() {
```

Comment:
Alternatively, if we want to include this in a 2.10.x patch release, but not backport it, we should use the 10210xx series.
```suggestion
function az_enterprise_attributes_import_update_1021001() {
```